### PR TITLE
Move instructions to README

### PR DESCRIPTION
**Pull request by esoergel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
...
+    ```
+  * To rebuild autostaging and push those changes to origin (necessary for deploy), run:
+     ```
+     $ scripts/rebuildstaging
+     ```
+* After rebuilding the autostaging branch, you need to deploy the new branch to staging.
+  ```
+  $ commcare-cloud --control staging deploy
+  ##### OR #####
+  $ scripts/rebuildstaging --deploy (NOTE: `commcare-cloud` must be available in your shell)
```
Comment (millerdev) [inline README.md:22]: Does this still work? I think I had trouble with it the last time I tried to run it.

**Comment by esoergel** [comment]
Merging now so devs see the updated README when they check the repo, but I'm still open to feedback on the changes (or just change it yourself if you feel like it, I'm not particular)

File: README.md [hunk 2]
```diff
@@ -1 +1,76 @@
...
+  ```
+
+## RESOLVING BRANCH CONFLICTS
+
+First, determine where the conflict lies.
```
Comment (millerdev) [inline README.md:27]: Would it make sense to add a note here about performing these git operations in the repo that has a conflict? That is, not in this _staging-branches_ repo.

File: README.md (hunk 1 above)
Comment (esoergel) [reply README.md:22]: no idea, I've never tried it

File: README.md (hunk 2 above)
Comment (esoergel) [reply README.md:27]: oh yeah, good point, I'll edit the file directly

File: README.md (hunk 1 above)
Comment (gherceg) [reply README.md:22]: I _think_ it just requires `cchq` being available to your shell, but could be wrong. I haven't set that up so typically don't follow this workflow either.

**Pull request merged**